### PR TITLE
made it so "pane gui scale" doesnt effect armour/pet huds

### PR DESCRIPTION
**Pull request by NopoTheGamer**
this did NOT take me 20 minutes to make this commit on the right branch

### CODE REVIEW

File: src/main/java/io/github/moulberry/notenoughupdates/core/GuiElementTextField.java [hunk 1]
```diff
@@ -395,6 +396,55 @@ public void keyTyped(char typedChar, int keyCode) {
...
+										case 7:
+											masterStarUnicode = "\u2791";
+											break;
+										case 8:
+											masterStarUnicode = "\u2792";
+											break;
+										case 9:
+											masterStarUnicode = "\u2793";
+											break;
+									}
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/core/GuiElementTextField.java:436]: I wish enhanced switch statements were in java 8

Comment (NopoTheGamer) [reply src/main/java/io/github/moulberry/notenoughupdates/core/GuiElementTextField.java:436]: uh?
 Unresolved conversations
1 conversation must be resolved before merging.

Comment (IRONM00N) [reply src/main/java/io/github/moulberry/notenoughupdates/core/GuiElementTextField.java:436]: didn't ask

**Comment by NopoTheGamer** [comment]
1 conflicting file 2.1.md

**Review by romangraef — DISMISSED** [review]
lgtm

**Review by DeDiamondPro — APPROVED** [review]
Very cool

**Review by romangraef — APPROVED** [review]
very cool

**Review by Lulonaut — APPROVED** [review]

**Pull request merged**